src/encoded/commands/deploy.py

```diff
@@ -789,54 +802,25 @@ def main():
-        if main_args.build_ami:
-            print('AMI Build: Demo deploying:', instance_info['name'])
-            print('instance_id:', instance_info['instance_id'])
-            print(
-                'After it builds, create the ami: '
-                "python ./cloud-config/create-ami.py {} demo {} --profile-name {}".format(
-                    instances_tag_data['username'],
-                    instance_info['instance_id'],
-                    main_args.profile_name,
-                )
-            )
-        else:
-            print('Deploying Demo({}): {}'.format(
-                instance_info['private_ip'],
-                instance_info['url']
-            ))
-            print(" ssh ubuntu@{}".format(instance_info['instance_id_domain']))
+        print('Deploying Demo({}): {}'.format(
+            instance_info['private_ip'],
+            instance_info['url']
+        ))
+        print(" ssh ubuntu@{}".format(instance_info['instance_id_domain']))
         print("ssh and tail:\n ssh ubuntu@{}{}".format(instance_info['public_dns'], tail_cmd))
     elif 'cluster_master' in instances_info and main_args.es_wait:
         instance_info = instances_info['cluster_master']
-        if main_args.build_ami:
-            print('AMI Build: Wait ES cluster deploying:', instance_info['name'])
-            print('instance_id:', instance_info['instance_id'])
-            arg_name = 'es-wait-head'
-            if main_args.es_elect:
-                arg_name = 'es-elect'
-            print(
-                'After it builds, create the ami: '
-                "python ./cloud-config/create-ami.py {} {} {} --profile-name {}".format(
-                    instances_tag_data['username'],
-                    arg_name,
-                    instance_info['instance_id'],
-                    main_args.profile_name,
-                )
-            )
-        else:
-            print('Deploying Head ES Node({}): {}'.format(
-                instance_info['private_ip'],
-                instance_info['name']
-            ))
-            print(" ssh ubuntu@{}".format(instance_info['instance_id_domain']))
+        print('Deploying Head ES Node({}): {}'.format(
+            instance_info['private_ip'],
+            instance_info['name']
+        ))
+        print(" ssh ubuntu@{}".format(instance_info['instance_id_domain']))
         print('\nRun the following command to view es head deployment log.')
         print("ssh ubuntu@{}{}".format(instance_info['public_dns'], tail_cmd))
         print('')
         helper_vars.append("datam='{}'".format(instance_info['instance_id']))
         for index in range(main_args.cluster_size):
-            str_index = str(index)
-            key_name = 'cluster_node_' + str_index
+            key_name = 'cluster_node_{}'.format(index)
             node_info = instances_info[key_name]
             helper_vars.append("data{}='{}'  # {}".format(index, node_info['instance_id'], key_name))
             if index == 0:
```

Comment:
Found a stale reference to `main_args.build_ami` below this, probably should remove also?

https://github.com/ENCODE-DCC/encoded/blob/61f82f1daa4a86c25b7deecb99605a0291229706/src/encoded/commands/deploy.py#L827

Reply:
`main_args.build_ami` means that after making the ES machine, it will be saved as an AMI. I don't think this is made any more stale by this PR. I agree that it is most likely not useful, but I would prefer to leave cleaning this up to the "official clean up ticket" that will follow after we confirm the that this way of deploying works.

Reply:
Ok just seemed a little inconsistent with removing similar conditionals from the single node deployments, I'm fine with leaving it for a subsequent PR.